_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -181,6 +185,14 @@ The `aria-orientation` property may not be used on `audio` element, nor it can b
 <audio src="/test-assets/moon-audio/moon-speech.mp3" controls aria-orientation="horizontal"></audio>
 ```
 
+#### Failed Example 3
+
+The `aria-label` property may not be used on an element with a `generic` role as it is [prohibited from being named](https://www.w3.org/TR/wai-aria-1.2/#namefromprohibited).
```

Comment:
Since the expectation is in term of "prohibited" attributes rather than roles "prohibited from being named", I'd rather have the example description using the same language too.